docs/architecture/adr-053-state-sync-prototype.md

```diff
@@ -0,0 +1,196 @@
+# ADR 053: State Sync Prototype
+
+This ADR describes an initial state sync prototype, and is subject to change as we gain feedback and experience.
```

Comment:
You might mention right off the bat how this is different from ADR 42.

Reply:
Good idea, done.